### DISCUSSION TITLE
Which 🇪🇺 country replaced the drachma with the euro?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#845] [Galaxy Trivia] [answered]
**Bonus:** What currency did 🇵🇹 use before the euro?

**Comment by peterbaccaro** [comment, marked as answer]
**Greece** replaced the drachma with the euro as its official currency. The transition took place on January 1, 2002, when Greece adopted the euro as part of the European Union (EU) and joined the Eurozone.

Before adopting the euro, Portugal used the **Portuguese escudo** (PTE) as its currency. The escudo was in circulation from 1911 until it was replaced by the euro on January 1, 2002, when Portugal joined the Eurozone.

**Comment by hasanerzi** [comment]
Greece. 

Greece adopted the euro as its official currency on January 1, 2001.

Bonus : Portugal used the Portuguese escudo (PTE) as its currency before adopting the euro. The escudo was replaced by the euro on January 1, 2002.
